<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyCloud</class>
 <widget class="QWidget" name="EasyCloud">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyCloud</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img/img/window_icon.jpg</normaloff>:/img/img/window_icon.jpg</iconset>
  </property>
  <widget class="QLabel" name="labelTitleBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(236, 65, 65)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelIcon">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>16</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/img/img/bar_icon.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelAppName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>18</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/img/img/bar_font.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMax">
   <property name="geometry">
    <rect>
     <x>944</x>
     <y>18</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButtonMax
{
	background-color:rgb(235, 65, 65);
	color:rgb(255, 222, 244);
	border:0px
}

QPushButton#pushButtonMax:hover
{
	background-color:rgb(235, 65, 65);
	color:white;
	border:0px
}</string>
   </property>
   <property name="text">
    <string>□</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>18</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButtonClose
{
	background-color:rgb(235, 65, 65);
	color:rgb(255, 222, 244);
	border:0px
}

QPushButton#pushButtonClose:hover
{
	background-color:rgb(235, 65, 65);
	color:white;
	border:0px
}</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMin">
   <property name="geometry">
    <rect>
     <x>904</x>
     <y>20</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButtonMin
{
	background-color:rgb(235, 65, 65);
	color:rgb(255, 222, 244);
	border:0px
}

QPushButton#pushButtonMin:hover
{
	background-color:rgb(235, 65, 65);
	color:white;
	border:0px
}</string>
   </property>
   <property name="text">
    <string>—</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>821</width>
     <height>541</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="labelPage1">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>0</y>
       <width>820</width>
       <height>540</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelCarousel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>784</width>
       <height>250</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/img/img/carousel.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="labelRecommend">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>260</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;</string>
     </property>
     <property name="text">
      <string>推荐歌曲 &gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRandom">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>320</y>
       <width>780</width>
       <height>194</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/img/img/random.png);
border:0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="labelPage2">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>0</y>
       <width>820</width>
       <height>540</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddLocalMusic">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>120</y>
       <width>151</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButtonAddLocalMusic
{
	background-color:rgb(236, 65, 65);
	border:0px;
	border-radius:18px;
	color:white;

}

QPushButton#pushButtonAddLocalMusic:hover
{
	background-color:rgb(205, 50, 50);
	border:0px;
	border-radius:18px;
	color:white;

}</string>
     </property>
     <property name="text">
      <string> 添加本地音乐</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMyPlayList">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;</string>
     </property>
     <property name="text">
      <string>我的播放列表</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlayListImg">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>93</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/img/img/playList.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonListPlay">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>121</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButtonListPlay
{
	background-color:white;
	border:1px solid rgb(224,224,224);
	border-radius:18px;
	color:rgb(51, 51, 51);

}

QPushButton#pushButtonListPlay:hover
{
	background-color:rgb(246, 246, 247);
	border:1px solid rgb(224,224,224);
	border-radius:18px;
	color:rgb(51, 51, 51);

}</string>
     </property>
     <property name="text">
      <string>开始播放</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetPlayList">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>180</y>
       <width>761</width>
       <height>351</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="labelPage3">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>0</y>
       <width>820</width>
       <height>540</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelSearch">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;</string>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSearchContent">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>701</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPlaySearchMusic">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>151</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButtonPlaySearchMusic
{
	background-color:rgb(236, 65, 65);
	border:0px;
	border-radius:18px;
	color:white;

}

QPushButton#pushButtonPlaySearchMusic:hover
{
	background-color:rgb(205, 50, 50);
	border:0px;
	border-radius:18px;
	color:white;

}</string>
     </property>
     <property name="text">
      <string>开始播放</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetSearchList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>761</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="labelPage4">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>0</y>
       <width>820</width>
       <height>540</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelRank">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;</string>
     </property>
     <property name="text">
      <string>官方榜</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelrankimg">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>185</width>
       <height>409</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/img/img/ranking.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetRankingList">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>115</y>
       <width>561</width>
       <height>398</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButtonDiscoverMusic">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButtonDiscoverMusic
{
	background-color:white;
	border:0px;
	border-radius:5px;
	color:rgb(51, 51, 51);
	text-align:left;
}

QPushButton#pushButtonDiscoverMusic:hover
{
	background-color:rgb(246, 246, 247);
	border:0px;
	border-radius:5px;
	color:black;
	text-align:left;
}</string>
      </property>
      <property name="text">
       <string> 发现音乐</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonPlayList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButtonPlayList
{
	background-color:white;
	border:0px;
	border-radius:5px;
	color:rgb(51, 51, 51);
	text-align:left;
}

QPushButton#pushButtonPlayList:hover
{
	background-color:rgb(246, 246, 247);
	border:0px;
	border-radius:5px;
	color:black;
	text-align:left;
}</string>
      </property>
      <property name="text">
       <string> 播放列表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSearchMusic">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButtonSearchMusic
{
	background-color:white;
	border:0px;
	border-radius:5px;
	color:rgb(51, 51, 51);
	text-align:left;
}

QPushButton#pushButtonSearchMusic:hover
{
	background-color:rgb(246, 246, 247);
	border:0px;
	border-radius:5px;
	color:black;
	text-align:left;
}</string>
      </property>
      <property name="text">
       <string> 搜索音乐</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonOnlineMusic">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButtonOnlineMusic
{
	background-color:white;
	border:0px;
	border-radius:5px;
	color:rgb(51, 51, 51);
	text-align:left;
}

QPushButton#pushButtonOnlineMusic:hover
{
	background-color:rgb(246, 246, 247);
	border:0px;
	border-radius:5px;
	color:black;
	text-align:left;
}</string>
      </property>
      <property name="text">
       <string> 官方榜</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelSideBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>201</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border-right:1px solid rgb(224,224,224);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>1021</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border-top:1px solid rgb(224,224,224);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>650</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::handle:horizontal 
{
	background-color: rgb(236,65,65);
}</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLastSong">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>615</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/img/img/lastSong.png);
border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>605</y>
     <width>50</width>
     <height>45</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/img/img/stop.png);
border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNextSong">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>615</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/img/img/nextSong.png);
border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSongImg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/img/img/window_icon.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelSongName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>610</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelArtist">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>640</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExpand">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>610</y>
     <width>59</width>
     <height>59</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/img/img/expand.png);
border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrentTime">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>648</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(153,153,153);</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>648</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(153,153,153);</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUserName">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>15</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>15</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>请输入歌曲名称</string>
   </property>
  </widget>
  <zorder>labelSideBar</zorder>
  <zorder>labelTitleBar</zorder>
  <zorder>labelIcon</zorder>
  <zorder>labelAppName</zorder>
  <zorder>pushButtonMax</zorder>
  <zorder>pushButtonClose</zorder>
  <zorder>pushButtonMin</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>labelPlayBar</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>pushButtonLastSong</zorder>
  <zorder>pushButtonPlay</zorder>
  <zorder>pushButtonNextSong</zorder>
  <zorder>labelSongImg</zorder>
  <zorder>labelSongName</zorder>
  <zorder>labelArtist</zorder>
  <zorder>pushButtonExpand</zorder>
  <zorder>labelCurrentTime</zorder>
  <zorder>labelTime</zorder>
  <zorder>labelUserName</zorder>
  <zorder>lineEdit</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
